src/z3c/authviewlet/tests/__init__.py

```diff
@@ -1,26 +1,27 @@
-##############################################################################
-#
-# Copyright (c) 2009 Zope Foundation and Contributors.
-# All Rights Reserved.
-#
-# This software is subject to the provisions of the Zope Public License,
-# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
-# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
-# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
-# FOR A PARTICULAR PURPOSE.
-#
-##############################################################################
-"""
-$Id:$
-"""
-
-import z3c.pagelet.browser
-
-
-class LoginLogoutPage(z3c.pagelet.browser.BrowserPagelet):
-    """Login and logout page."""
-
-    # make sure the pagelet renders no content, as this is not what we
-    # want to show with this pagelet
-    template = lambda self: ''
+##############################################################################
+#
+# Copyright (c) 2009 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""
+$Id:$
```

Comment:
What is going on here.  Did the file have DOS line endings?

(Also as long as we're changing everything, we might as well mass-delete the Subversion `$Id:$` placeholders.)

Reply:
(Yeah, the diff becomes nice and readable after asking GH to hide whitespace changes.  The point about `$Id:$` stands.)

Reply:
Yes the files actually had DOS line endings. I am going to remove the SVN leftovers.

Reply:
Hm, it was just this single one.